.github/workflows/run-integration-test-dispatch.yaml

```diff
@@ -0,0 +1,87 @@
+# Workflow Dispatch to run terra-ui integration tests
+# If trying to run terra-ui tests as part of CI from other GHA, use the run-integration-tests workflow_call instead.
+name: External terraui Test
+on:
+  workflow_dispatch:
+    inputs:
+
+      ##
+      ## Configurations:
+      ##
+
+      test_url:
+        required: false
+        type: string
+        description: "URL of terraui instance to test against, required"
```

Comment:
Why does the description say it's required if `required` is set to false?

Reply:
Also, given the trouble we had with trailing slashes, might be nice to add an example. Though for the trailing slashes, I think we should update the tests to handle those.

Reply:
updated `required` to true
added example